lib/ui/index.ts

```diff
@@ -138,9 +123,12 @@ const uiBetaTag = (message, log = true) => {
   } else {
     return result;
   }
-};
+}
 
-const uiDeprecatedTag = (message, log = true) => {
+export function uiDeprecatedTag(
+  message: string,
+  log: boolean = true
+): void | string {
```

Comment:
It doesn't seem like the return value for this function is being used anywhere so can probably just remove it

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1233/commits/3ec8a544996c20de5a030ed8db00991e855fe90d